packages/core/src/runtime/compileCSS.ts

```diff
@@ -8,6 +8,7 @@ export interface CompileCSSOptions {
 
   pseudo: string;
   media: string;
+  layer: string;
```

Comment:
@layershifter Don't we also need to update https://github.com/microsoft/griffel/blob/bc30fac684d0a8f8a9d92f84847bf111b87b4f71/packages/core/src/renderer/rehydrateRendererCache.ts#L6
and corresponding UT (`rehydrateCache() avoids double insertion`)?

Reply:
added layer into line 6